src/app/components/ideas-list/ideas-list.component.ts

```diff
@@ -14,28 +14,44 @@ export class IdeasListComponent {
 
   showPublish = false;
   loading = false;
+  ideasAscending = false;
```

Comment:
Instead of having two variables you can create a type in this file outside of the class `export type SortOrder = 'ASC' | 'DESC' | ''` and have only one variable `sortOrder: SortOder = ''`. The sort type is not mandatory so the user can disabled both (sortOrder = '').